### ISSUE TITLE
Security accessibility of the overriding method

### ISSUE THREAD

**Issue by QVMaster** [Antaris/RazorEngine#449]
Hi, I don't have problem when using razorEngin on local IIS, but on deploying to remote host if try this way:
var service = RazorEngineService.Create(config);
Engine.Razor = service;
var emailHtmlBody = Engine.Razor.RunCompile(tmplfile, "tmpName", typeof(EmailTemplateViewModel), model, null);
I got this error:
Inheritance security rules violated while overriding member: 'RazorEngine.CrossAppDomainObject.InitializeLifetimeService()'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

and if not create service and just call as below:
var emailHtmlBody = Engine.Razor.RunCompile(tmplfile, "tmpName", typeof(EmailTemplateViewModel), model, null);
I got the error like this:
Inheritance security rules violated while overriding member: 'RazorEngine.CrossAppDomainObject.InitializeLifetimeService()'. Security accessibility of the overriding method must match the security accessibility of the method being overriden.

I think it happen because I didn't full trust access on remote server, but I don't know how can I use it to avoid issue, Can anyone help?
